ports/raspberrypi/boards/bradanlanestudio_explorer_rp2040/pins.c

```diff
@@ -111,17 +111,16 @@ static const mp_rom_map_elem_t board_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_TOUCH9), MP_ROM_PTR(&pin_GPIO27) },
 
     { MP_ROM_QSTR(MP_QSTR_GP28), MP_ROM_PTR(&pin_GPIO28) },
-    // GPIO28 is also the interrupt pin of the accelerometer
-    { MP_ROM_QSTR(MP_QSTR_ACCEL_INT), MP_ROM_PTR(&pin_GPIO28) },
+    // GPIO28 is also the interrupt pin of the accelerometer on some boards
```

Comment:
Did you mean to remove `{ MP_ROM_QSTR(MP_QSTR_ACCEL_INT), MP_ROM_PTR(&pin_GPIO28) },`? If so, then maybe remove the line 114 comment?